invenio_app_ils/circulation/utils.py

```diff
@@ -94,7 +94,7 @@ def circulation_overdue_loan_days(loan):
 def circulation_upcoming_return_range():
     """Return a default upcoming return range."""
     return arrow.utcnow() + timedelta(
-        days=current_app.config["ILS_UPCOMING_RETURN_RANGE"])
+        days=current_app.config["ILS_LOAN_OVERDUE_DAYS_UPFRONT_NOTIFICATION"])
```

Comment:
`ILS_LOAN_OVERDUE_DAYS_UPFRONT_NOTIFICATION` is ridiculous, this is a good reason to remain as is `upcoming_return_range` and add a better explanation in the `config.py`

Reply:
Or `ILS_OVERDUE_LOANS_FILTER_RANGE` plus better explanation in the `config.py`? :)

Reply:
@ntarocco I got it meant to mean but I just checked that in the end we are using it in the overdue facet to just say "Check all loans from now until than date" where that date is the value of the `circulation_upcoming_return_range `. So I just proposed to simplify the name if it will be used only there...But not strong opinion on this!